src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx

```diff
@@ -258,72 +258,58 @@ export const ViewEvaluationHeader = () => {
         </pre>
       </div>
       <h2 className='mt-5 mb-5 text-2xl font-bold'>Employees</h2>
-      <Dialog open={showCancelDialog}>
-        <Dialog.Title>Cancel Evaluation</Dialog.Title>
-        <Dialog.Description>
-          Are you sure you want to cancel this record? <br /> This action cannot be reverted.
-        </Dialog.Description>
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={toggleCancelDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handleCancel}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
-      <Dialog open={showDeleteDialog}>
-        <Dialog.Title>Delete Evaluation</Dialog.Title>
-        <Dialog.Description>
-          Are you sure you want to delete this record? <br /> This action cannot be reverted.
-        </Dialog.Description>
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={toggleDeleteDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handleDelete}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
-      <Dialog open={showCloseDialog}>
-        <Dialog.Title>Close Evaluation</Dialog.Title>
-        <Dialog.Description>
-          Are you sure you want to close this record? <br /> This action cannot be reverted.
-        </Dialog.Description>
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={toggleCloseDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handleClose}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
-      <Dialog open={showPublishDialog}>
-        <Dialog.Title>Publish Evaluation</Dialog.Title>
-        <Dialog.Description>Are you sure you want to publish this record?</Dialog.Description>
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={togglePublishDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handlePublish} loading={loading === Loading.Pending}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
-      <Dialog open={showReopenDialog}>
-        <Dialog.Title>Reopen Evaluation</Dialog.Title>
-        <Dialog.Description>Are you sure you want to reopen this record?</Dialog.Description>
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={toggleReopenDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handleReopen} loading={loading === Loading.Pending}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
+      <CustomDialog
```

Comment:
showSubmitButton and showCloseButton props not passed, same for other dialogs

Reply:
I set default values for `showSubmitButton` and `showCloseButton` to true inside the CustomDialog component so the buttons would be visible by default. Is that okay?

Reply:
sorry didnt notice, thanks catt! Approved

Reply:
Alright, will re-request review ate thanks so much!